src/migrations/20190705144236-create-user-table.ts

```diff
@@ -0,0 +1,32 @@
+import { DataTypes, QueryInterface } from "sequelize";
+
+export default {
+  up: (queryInterface: QueryInterface): Promise<void> => {
+    return queryInterface.createTable("User", {
```

Comment:
use plural for tables

```suggestion
    return queryInterface.createTable("Users", {
```